### ISSUE TITLE
"Always on top" not applied to browser access request or Auto-Type window  

### ISSUE THREAD

**Issue by kud** [keepassxreboot/keepassxc#9330]
## Overview
When you choose "always on top", the popup "browser access request" can be displayed but when you want to click on it, it's the main window which is used, and you can't make the request appear again.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Use "always on top"
2. Create a new password for a specific website
3. Trigger the browser access request

## Expected Behavior
browser access request window also on top

## Actual Behavior
browser access request is less priorised than the main window

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - 2.7.4

[NOTE]: # ( Pick choices based on your environment )
Operating System: macOS

**Comment by droidmonkey** [comment]
Good catch, we will have to apply the always on top flag to this window and auto-type as well. This isn't just a macOS bug.

**Comment by ian-h-chamberlain** [comment]
Just a quick note as I also ran into this, (at least on macOS) it makes the main window unusable / unclickable, but you can still move the main window out of the way (e.g. onto another monitor, to the corner of the screen) to find the access request window and continue normally. A bit unintuitive but it is at least possible to work around.
